mods/default/player.lua

```diff
@@ -58,12 +59,14 @@ function default.player_set_model(player, model_name)
 			textures = player_textures[name] or model.textures,
 			visual = "mesh",
 			visual_size = model.visual_size or {x=1, y=1},
+			collisionbox = model.collisionbox or {-0.3, 0.0, -0.3, 0.3, 1.77, 0.3},
 		})
 		default.player_set_animation(player, "stand")
 	else
 		player:set_properties({
 			textures = { "player.png", "player_back.png", },
 			visual = "upright_sprite",
+			collisionbox = {-0.3, 0.0, -0.3, 0.3, 1.77, 0.3},
```

Comment:
Maybe leave this at 1.75? I have not tested how it looks, just guessing it would've be designed that way.